install-task-doc.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
Got the reason in the description of https://github.com/RackHD/RackHD/issues/316.